ENDPOINTS.md

```diff
@@ -35,10 +35,12 @@ GET /client/about||[**Client.about**](https://github.com/CiscoDevNet/vManage-cli
-POST /template/policy/definition/control||[**ConfigurationPolicyControlDefinition.create_policy_definition**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/policy/definition/control.py#L25)|[**ControlPolicy**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/models/policy/definitions/control.py#L299)|[**PolicyDefinitionId**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/models/policy/policy_definition.py#L1008)|
+POST /disasterrecovery/activate||[**ConfigurationDisasterRecovery.activate**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/disaster_recovery.py#L94)||[**ActivateDisasterRecoveryTaskId**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/disaster_recovery.py#L24)|
+POST /disasterrecovery/deregister||[**ConfigurationDisasterRecovery.delete**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/disaster_recovery.py#L98)||[**DeregisterDisasterRecoveryTaskId**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/disaster_recovery.py#L20)|
+GET /disasterrecovery/details||[**ConfigurationDisasterRecovery.get_details**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/disaster_recovery.py#L142)||[**DisasterRecoveryDetailsResponse**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/disaster_recovery.py#L38)|
+POST /disasterrecovery/validateNodes||[**ConfigurationDisasterRecovery.get_reachability_info**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/disaster_recovery.py#L166)|list[[**ValidateNodePayload**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/disaster_recovery.py#L42)]|DataSequence[[**ValidatedNodeEntry**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/disaster_recovery.py#L48)]|
+POST /disasterrecovery/pause||[**ConfigurationDisasterRecovery.pause_dr**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/disaster_recovery.py#L186)||[**DisasterRecoveryPauseResponse**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/disaster_recovery.py#L53)|
+POST /disasterrecovery/register||[**ConfigurationDisasterRecovery.register**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/disaster_recovery.py#L202)|[**DisasterRecoveryRegisterPayload**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/disaster_recovery.py#L83)|[**RegisterDisasterRecoveryTaskId**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/disaster_recovery.py#L16)|
```

Comment:
This github links return 404

Reply:
Ok will fix

Reply:
The links point to main branch they WILL work after PR is merged
However this is another reason not to update them on each PR, but rather for new release (we will fix that in future)

Reply:
Created issue: https://github.com/CiscoDevNet/vManage-client/issues/437
We will adress that in another PR